nncf/common/pruning/utils.py

```diff
@@ -391,3 +394,18 @@ def get_operator_metatype_by_op_name(self, op_name: str):
         if op_name in self._op_name_to_op_class:
             return self._op_name_to_op_class[op_name]
         return None
+
+
+def is_depthwise_conv(node: NNCFNode) -> bool:
+    return isinstance(node.layer_attributes, ConvolutionLayerAttributes) \
+           and node.layer_attributes.groups == node.layer_attributes.in_channels \
+           and (node.layer_attributes.out_channels % node.layer_attributes.in_channels == 0) \
+           and node.layer_attributes.in_channels > 1
```

Comment:
I guess you can implement it something like that:
```
return node.metatype in DEPTWISE_CONVOLUTION
```
or
```
return DepthwiseConvolution.subtype(node.metatype)
```

Reply:
I'm afraid to implement this kind of grouping for depthwise-ness just now. Propose to leave it as is.